test_package/conanfile.py

```diff
@@ -7,7 +7,7 @@
 
 class TestPackageConan(ConanFile):
     settings = "os", "compiler", "build_type", "arch"
-    generators = "qt", "cmake"
+    generators = "qt", "cmake", "cmake_find_package_multi"
```

Comment:
BTW: Does/Should recipe (not test_package recipe) contain `generators` member?

Reply:
it already has: https://github.com/bincrafters/conan-qt/blob/testing/5.15.1/conanfile.py#L45, the point is that the pkg-config files are needed to build qt itself. There is no need to add any cmake generators in the recipe itself.